sympy/solvers/ode/ode.py

```diff
@@ -731,6 +731,216 @@ def _helper_simplify(eq, hint, match, simplify=True, ics=None, **kwargs):
             rv = rv1
     return rv
 
+def dsubs(eq, trans):
+    r"""
+    Function that substitutes for variables in an Ordinary Differential Equation.
+
+    The inputs are -
+
+    ``eq``: The differential equation which is to be transformed.
+
+    ``trans``: It is a tuple of 2 elements containing -
+
+        a. ``forward_trans``: A dictionary specifying the variable transformations
+        with the old variable as key and the transformation as value.
+
+        The second element of the tuple can be either of the following -
+
+        b. ``reverse_trans``: A dictionary specifying the variable transformations
```

Comment:
This isn't actually used in the code anywhere. I would omit it, at least for now. If it ends up being used later, we can add it back in.